### ISSUE TITLE
The 'simple' example crashes when running

### ISSUE THREAD

**Issue by ShenMian** [projectharmonia/bevy_enhanced_input#10]
```console
... SKIP ...
   Compiling bevy_enhanced_input v0.4.0 (D:\Users\sms\Desktop\bevy_enhanced_input)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 1m 09s
     Running `target\debug\examples\simple.exe`
2024-11-30T22:39:54.992973Z  INFO bevy_render::renderer: AdapterInfo { name: "NVIDIA GeForce RTX 4060 Laptop GPU", vendor: 4318, device: 10464, device_type: DiscreteGpu, driver: "NVIDIA", driver_info: "566.14", backend: Vulkan }
thread 'main' panicked at D:\apps\scoop\persist\rustup\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bevy_ecs-0.15.0\src\system\system_param.rs:911:17:
Resource requested by (bevy_ecs::system::commands::Commands, bevy_ecs::system::query::Query<(bevy_ecs::entity::Entity, &mut bevy_window::window::Window, core::option::Option<&bevy_window::raw_handle::RawHandleWrapperHolder>), bevy_ecs::query::filter::Added<bevy_window::window::Window>>, bevy_ecs::event::writer::EventWriter<bevy_window::event::WindowCreated>, bevy_ecs::change_detection::NonSendMut<bevy_winit::winit_windows::WinitWindows>, bevy_ecs::change_detection::NonSendMut<bevy_winit::accessibility::AccessKitAdapters>, bevy_ecs::change_detection::ResMut<bevy_winit::accessibility::WinitActionRequestHandlers>, bevy_ecs::change_detection::Res<bevy_a11y::AccessibilityRequested>, bevy_ecs::change_detection::Res<bevy_winit::winit_monitors::WinitMonitors>) does not exist: bevy_ecs::event::collections::Events<bevy_window::event::WindowCreated>
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\examples\simple.exe` (exit code: 101)
```

**Comment by Shatur** [comment]
Can confirm!
This is weird, happens right after upgrading to 0.15.0. Works just fine on 045bbf7eabbf023b5864c411d0c6d2cddc7cb943. In the meanwhile you can try this commit, I need to read Bevy changelog first... Looks like it happens in a Bevy system.

**Comment by Shatur** [comment]
Fixed, apparently 0.15.0 needs more dev-dependencies.
